### ISSUE TITLE
Commands/commands history - nothing happens after <CR> or Ctrl+E

### ISSUE THREAD

**Issue by tamirzb** [ibhagwan/fzf-lua#258]
Perhaps I'm missing something, but it seems that both `commands` and `commands_history` are not working for me. I see the commands/commands history, but when I press either \<CR\> or Ctrl+E the FZF window disappears but nothing happens.

Any idea what's wrong here?

**Comment by ibhagwan** [comment]
Not sure what's wrong here but we can easily test that, you can redefine the `<CR>` action  with the same code and see where it fails:

You can define a new function and call it instead, like so:

```lua
_G.test_cmd = function()
  local opts = {
    actions ={
      ["default"] = function(selected)
        print("command :", selected[1])
        vim.fn.feedkeys(string.format(":%s", selected[1]))
      end
    }
  }
  require'fzf-lua'.command_history(opts)
end
```

Then run it (as shown below) and see you get printout of the command in `:messages`?
```lua
:lua _G.test_cmd()
```

**Comment by tamirzb** [comment]
After looking at your test code and debugging a bit further I figured out what the issue is. By default `vim.fn.feedkeys` remaps keys. As I have a mapping for `:` that mapping was used.

I opened a PR to fix this (#259) which adds the `"n"` argument to `feedkeys`, which means it would ignore these mappings.

**Comment by ibhagwan** [comment]
You're correct, tysm for the contribution, merged!
